configuration_controller/mappings/request_mapping.yml

```diff
@@ -0,0 +1,19 @@
+---
+registrationRequest:
+  sas_endpoint: registration
+  http_method: post
+spectrumInquiryRequest:
+  sas_endpoint: spectrumInquiry
+  http_method: post
+grantRequest:
+  sas_endpoint: grant
+  http_method: post
+heartbeatRequest:
+  sas_endpoint: heartbeat
+  http_method: post
+relinquishmentRequest:
+  sas_endpoint: relinquishment
+  http_method: post
+deregistrationRequest:
+  sas_endpoint: deregistration
+  http_method: post
```

Comment:
Marcin has chosen convention of naming YAML files *.yaml* in deployment tools. Maybe let's stick to one convention across the whole project.

Reply:
Marcin will create a PR changin his extensions to yml